src/backup_entity_migration/backup_entity_migration_handlers.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+import webapp2
+from google.appengine.ext import ndb
+from google.appengine.ext.ndb import Cursor
+
+from src.backup.datastore.Backup import Backup
+from src.backup.datastore.Table import Table
+from src.commons.config.configuration import configuration
+from src.commons.tasks import Tasks
+
+
+class BackupEntityMigrationHandler(webapp2.RequestHandler):
+
+    def get(self):
+
+        table_cursor_value = self.request.get('cursor', None)
+        if table_cursor_value:
+            table_cursor = Cursor(urlsafe=table_cursor_value)
+        else:
+            table_cursor = None
+        logging.info("Migration of Backup entities was started. Table_cursor: %s",
+                     table_cursor)
+
+        self.schedule_backup_entities(table_cursor)
+
+    def schedule_backup_entities(self, table_cursor):
+        tables, cursor, more = Table.get_all_with_cursor(table_cursor)
+
+        Tasks.schedule('entity-backup-migration-worker',
+                       self.__create_table_backups_migration_tasks(tables))
+
+        if more:
```

Comment:
This should be:
`if more and next_cursor:`
See https://cloud.google.com/appengine/docs/standard/python/ndb/queries#cursors